src/test/java/com/wikia/webdriver/testcases/notificationstests/messagewallnotifications/MessageWallNotificationsFollowersMessageTests.java

```diff
@@ -59,7 +60,7 @@ public void followerNotificationNewMessage_setup_2() {
     wall.writeTitle(title);
     wall.submit();
     wall.verifyMessageText(title, message, credentials.userName6);
-  }
+  }}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)